extensions/registration-service/src/main/java/org/eclipse/dataspaceconnector/registration/credential/VerifiableCredentialService.java

```diff
@@ -0,0 +1,22 @@
+/*
+ *  Copyright (c) 2022 Microsoft Corporation
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Microsoft Corporation - initial API and implementation
+ *
+ */
+
+package org.eclipse.dataspaceconnector.registration.credential;
+
+import org.eclipse.dataspaceconnector.registration.authority.model.Participant;
+import org.eclipse.dataspaceconnector.spi.response.StatusResult;
+
+public interface VerifiableCredentialService {
+    StatusResult<Void> publishVerifiableCredential(Participant participant);
```

Comment:
I think `pushVerifiableCredential` would be a better name as IMO publish implies to make it public somewhere (which it won't be when IH has authentication in place)